src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
What about having two delimiters one for checksumed entries and another one for non-checksumed entries. That way we don't have to use the extra 4 bytes to zero out the checksum.

Also, what is the use case here? is it just for perf testing (i.e. determining overhead)?

Mixing both checksumed and non-checksumed entries is error prone and doesn't seem to
be practical.